aptos-move/framework/src/natives/cryptography/bls12381.rs

```diff
@@ -685,7 +735,16 @@ pub fn make_all(gas_params: GasParameters) -> impl Iterator<Item = (String, Nati
             "verify_signature_share_internal",
             make_native_from_func(gas_params, native_bls12381_verify_signature_share),
         ),
-    ];
-
+    ]);
+    natives.append(&mut vec![
```

Comment:
I think you need to gate this by the "testing" feature flag.

Reply:
Yep.